src/intersections.jl

```diff
@@ -62,6 +62,13 @@ Type `IntersectionType` in a Julia session to see the full list.
   CrossingLines
   OverlappingLines
 
+  # ray-ray intersection
+  CrossingRays
+  MidTouchingRays
+  CornerTouchingRays
+  OverlappingAgreeingRays
+  OverlappingOpposingRays
+
```

Comment:
Please move this section to after the segment-segment section so that we track the difference over time

Reply:
Sure :)